package com.wposs.catalogo.dto;

import java.math.BigDecimal;

public record ProductoResumen(
        Long id,
        String titulo,
        BigDecimal precio
) {
}